packages/sdk/test/loyalties.spec.ts

```diff
@@ -192,6 +193,12 @@ describe('Loyalties API', () => {
 		} as LoyaltiesListCardTransactionsResponseBody)
 	})
 
+	it('Should list member activity', async () => {
+		const listMemberActivityResponse = await client.loyalties.listMemberActivity(null, loyaltiesMember.code)
+		expect(listMemberActivityResponse.data.length).toBeGreaterThan(0)
+		expect(typeof listMemberActivityResponse.data[0].type).toEqual('string')
```

Comment:
I'm wondering if checking this `type` is enough
there are 4 other properties that could be checked somehow
I know this is not a place for testing precisely everything - we just need to confirm this endpoint works in the SDK, so I suppose this is okey already
but still... maybe checking these 4 other properties (just their types) would make this test a bit better with little effort?

Reply:
`data` is tagged as `any`, checking if a 1 property exists is sufficient to know that the rest should be recognized ass well.
Of course in our case this most likely would be a same event oven and over again. 
But what if sometime in the future Voucherify will not create this event but a different one during loyalty card publication. Tests will fail.

Reply:
sure, ok
and thanks for the change, more `expect`s is indeed what I had in mind :)